shared/mocks.js

```diff
@@ -34,5 +35,7 @@ module.exports = {
     [/route-tree$/, '/../__mocks__/empty'],
     [/typed-connect/, '/../util/__mocks__/typed-connect'],
     [/util\/saga/, '/../__mocks__/saga'],
+    // TODO: this is not compatible with both mock places
```

Comment:
?

Reply:
webpack.config.js specifies the locations of the mocks with respect to webpack.config.js, whereas metro.config.js specifies the locations of the mocks with respect to the files themselves. Will fix.